### PR TITLE
remove duplicated charts generated by the new chart-build-script

### PR DESCRIPTION
**Pull request by jiaqiluo**
issue: https://github.com/rancher/rancher/issues/32842

Here is the list of charts being removed. The contents of those charts are identical to the related ones except the versions. 

```
- fleet 0.3.500+up0.3.5 
- fleet-crd 0.3.500+up0.3.5 
- fleet-agent 0.3.500+up0.3.5 

- longhorn 1.1.100+up1.1.1
- longhorn-crd 1.1.100+up1.1.1

- rancher-gatekeeper 3.3.1+up3.3.0
- rancher-gatekeeper-crd 3.3.1+up3.3.0

- rancher-logging 3.9.400+up3.9.4
- rancher-logging-crd 3.9.400+up3.9.4

- rancher-operator 0.1.400+up0.1.4 
- rancher-operator-crd 0.1.400+up0.1.4 

- rancher-vsphere-cpi 1.0.0 
- rancher-vsphere-csi 2.1.0 

- rancher-external-ip-webhook 0.1.601+up0.1.6 
- rancher-wins-upgrader 0.0.100+up0.0.1 
```

the `doNotRelease: true` filed is added to some of their `package.yaml` for the CI to be passed. 
A following PR will be made to remove all those  `doNotRelease: true` filed

### CODE REVIEW

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
CI passing implies `doNotTrack` was added to all the appropriate packages. Checking the following:

- fleet 0.3.500+up0.3.5
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- fleet-crd 0.3.500+up0.3.5 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- fleet-agent 0.3.500+up0.3.5 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- longhorn 1.1.100+up1.1.1
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- longhorn-crd 1.1.100+up1.1.1
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-external-ip-webhook 0.1.601+up0.1.6 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-gatekeeper 3.3.1+up3.3.0
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-gatekeeper-crd 3.3.1+up3.3.0
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-logging 3.9.400+up3.9.4
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-logging-crd 3.9.400+up3.9.4
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-operator 0.1.400+up0.1.4 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-operator-crd 0.1.400+up0.1.4 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-vsphere-cpi 1.0.0 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-vsphere-csi 2.1.0 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed
- rancher-wins-upgrader 0.0.100+up0.0.1 
  - [x] chart removed
  - [x] asset removed
  - [x] index.yaml entry removed

---

Ran the following command to inspect output and ensure all charts that have duplicate versions are cleaned up:

```bash
$ curl https://raw.githubusercontent.com/jiaqiluo/charts/cleanup-25/index.yaml | yq e '.entries[] | to_entries | .[].value | {.name: .version}' -
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  148k  100  148k    0     0   511k      0 --:--:-- --:--:-- --:--:--  511k
fleet: 0.3.500
fleet: 0.3.400
fleet: 0.3.300
fleet: 0.3.200
fleet: 0.3.100
fleet: 0.3.000
fleet-agent: 0.3.500
fleet-agent: 0.3.400
fleet-agent: 0.3.300
fleet-agent: 0.3.200
fleet-agent: 0.3.100
fleet-agent: 0.3.000
fleet-crd: 0.3.500
fleet-crd: 0.3.400
fleet-crd: 0.3.300
fleet-crd: 0.3.200
fleet-crd: 0.3.100
fleet-crd: 0.3.000
longhorn: 1.1.200+up1.1.2
longhorn: 1.1.100
longhorn: 1.1.001
longhorn: 1.1.000
longhorn: 1.0.201
longhorn: 1.0.200
longhorn-crd: 1.1.200+up1.1.2
longhorn-crd: 1.1.100
longhorn-crd: 1.1.001
longhorn-crd: 1.1.000
longhorn-crd: 1.0.201
longhorn-crd: 1.0.200
rancher-alerting-drivers: 1.0.200
rancher-alerting-drivers: 1.0.100
rancher-backup: 1.2.0-rc3
rancher-backup: 1.0.400
rancher-backup: 1.0.301
rancher-backup: 1.0.300
rancher-backup: 1.0.201
rancher-backup: 1.0.200
rancher-backup-crd: 1.2.0-rc3
rancher-backup-crd: 1.0.400
rancher-backup-crd: 1.0.301
rancher-backup-crd: 1.0.300
rancher-backup-crd: 1.0.201
rancher-backup-crd: 1.0.200
rancher-cis-benchmark: 1.0.400
rancher-cis-benchmark: 1.0.301
rancher-cis-benchmark: 1.0.300
rancher-cis-benchmark: 1.0.200
rancher-cis-benchmark: 1.0.100
rancher-cis-benchmark-crd: 1.0.400
rancher-cis-benchmark-crd: 1.0.301
rancher-cis-benchmark-crd: 1.0.300
rancher-cis-benchmark-crd: 1.0.200
rancher-cis-benchmark-crd: 1.0.100
rancher-external-ip-webhook: 0.1.601
rancher-external-ip-webhook: 0.1.600
rancher-external-ip-webhook: 0.1.400
rancher-gatekeeper: 3.3.001
rancher-gatekeeper: 3.3.000
rancher-gatekeeper: 3.2.101
rancher-gatekeeper: 3.1.101
rancher-gatekeeper: 3.1.100
rancher-gatekeeper-crd: 3.3.001
rancher-gatekeeper-crd: 3.3.000
rancher-gatekeeper-crd: 3.2.101
rancher-gatekeeper-crd: 3.1.101
rancher-gatekeeper-crd: 3.1.100
rancher-grafana: 6.11.0
rancher-grafana: 6.6.401
rancher-istio: 1.9.800
rancher-istio: 1.9.600
rancher-istio: 1.9.500
rancher-istio: 1.9.300
rancher-istio: 1.8.600
rancher-istio: 1.8.500
rancher-istio: 1.8.300
rancher-istio: 1.7.301
rancher-istio: 1.7.300
rancher-istio: 1.7.100
rancher-kiali-server: 1.32.100
rancher-kiali-server: 1.29.000
rancher-kiali-server: 1.24.003
rancher-kiali-server: 1.24.001
rancher-kiali-server: 1.23.001
rancher-kiali-server-crd: 1.32.100
rancher-kiali-server-crd: 1.29.000
rancher-kiali-server-crd: 1.24.003
rancher-kiali-server-crd: 1.24.001
rancher-kiali-server-crd: 1.23.001
rancher-kube-state-metrics: 3.2.0
rancher-kube-state-metrics: 2.13.101
rancher-logging: 3.14.0
rancher-logging: 3.9.400
rancher-logging: 3.9.000
rancher-logging: 3.8.201
rancher-logging: 3.6.001
rancher-logging: 3.6.000
rancher-logging-crd: 3.14.0
rancher-logging-crd: 3.9.400
rancher-logging-crd: 3.9.000
rancher-logging-crd: 3.8.201
rancher-logging-crd: 3.6.001
rancher-logging-crd: 3.6.000
rancher-monitoring: 16.6.0
rancher-monitoring: 14.5.100
rancher-monitoring: 9.4.203
rancher-monitoring: 9.4.202
rancher-monitoring: 9.4.201
rancher-monitoring: 9.4.200
rancher-monitoring-crd: 16.6.0
rancher-monitoring-crd: 14.5.100
rancher-monitoring-crd: 9.4.203
rancher-monitoring-crd: 9.4.202
rancher-monitoring-crd: 9.4.201
rancher-monitoring-crd: 9.4.200
rancher-node-exporter: 1.18.100+up1.18.1
rancher-node-exporter: 1.16.201
rancher-operator: 0.1.400
rancher-operator: 0.1.300
rancher-operator: 0.1.200
rancher-operator: 0.1.100
rancher-operator: 0.1.000
rancher-operator-crd: 0.1.400
rancher-operator-crd: 0.1.300
rancher-operator-crd: 0.1.200
rancher-operator-crd: 0.1.100
rancher-operator-crd: 0.1.000
rancher-prom2teams: 0.2.100+up0.2.0
rancher-prom2teams: 0.2.0
rancher-prom2teams: 0.2.000
rancher-prometheus-adapter: 2.14.0
rancher-prometheus-adapter: 2.12.101
rancher-pushprox: 0.1.400
rancher-pushprox: 0.1.300
rancher-pushprox: 0.1.201
rancher-pushprox: 0.1.2
rancher-pushprox: 0.1.1
rancher-pushprox: 0.1.0
rancher-sachet: 1.0.200
rancher-sachet: 1.0.100
rancher-tracing: 1.20.100
rancher-tracing: 1.20.002
rancher-tracing: 1.20.001
rancher-vsphere-cpi: 1.0.000
rancher-vsphere-csi: 2.1.000
rancher-webhook: 0.1.200-rc1+up0.1.2-rc1
rancher-webhook: 0.1.100+up0.1.1
rancher-webhook: 0.1.000
rancher-webhook: 0.1.0-beta901
rancher-webhook: 0.1.0-beta900
rancher-webhook: 0.1.0-beta700
rancher-webhook: 0.1.0-beta500
rancher-windows-exporter: 0.1.100
rancher-windows-exporter: 0.1.000
rancher-wins-upgrader: 0.0.100
rio: 0.8.001
rio: 0.8.1
rio: 0.8.000
```

---

TLDR:

`rancher-prom2teams: 0.2.0` and `rio: 0.8.1` still need to be cleaned up since a released version already exists that tracks the same underlying version.

@jiaqiluo can you remove these in a separate commit (without changing the existing commits, which would invalidate my review)?

After those changes are made, this PR LGTM.

**Comment by jiaqiluo** [comment]
@aiyengar2  The chart `rancher-prom2teams: 0.2.0` and `rio: 0.8.1` are removed. Please check the latest 2 commits.

**Comment by brendarearden** [comment]
I am not seeing a `charts/` removed for this asset that was removed `assets/rancher-prom2teams/rancher-prom2teams-0.2.100+up0.2.0.tgz`

**Comment by jiaqiluo** [comment]
@brendarearden  `rancher-prom2teams 0.2.100+up0.2.0` is a new chart we will release. 
`rancher-prom2teams: 0.2.0` is the duplicated one that gets removed. 

```
diff -r charts/rancher-prom2teams/rancher-prom2teams/0.2.100+up0.2.0/Chart.yaml charts/rancher-prom2teams/rancher-prom2teams/0.2.000/Chart.yaml
10c10
< version: 0.2.100+up0.2.0
---
> version: 0.2.000
diff -r charts/rancher-prom2teams/rancher-prom2teams/0.2.100+up0.2.0/templates/psp.yaml charts/rancher-prom2teams/rancher-prom2teams/0.2.000/templates/psp.yaml
29d28
<     - 'secret'
```

**Review by brendarearden — APPROVED** [review]
lgtm!

**Review by aiyengar2 — APPROVED** [review]
LGTM, ship it! 🥳

**Review by cbron — APPROVED** [review]
lgtm, once this PR merges we should ensure nothing weird has happened, like charts getting regenerated or something.
